spec/features/home_page_spec.rb

```diff
@@ -2,13 +2,19 @@
 
 feature "Visiting the homepage", :js do
   scenario "has errors" do
+    allow(Rails.logger).to receive(:warn).and_call_original
+
     visit "/"
 
-    fill_in "my-input", with: "my text!"
-    error_messages = page.driver.error_messages.map do |message|
-      message.fetch(:message)
-    end
+    expect(page).to have_field "my-input"
+    expect(page).not_to have_css "label"
+    expect(page).not_to have_css "html[lang]"
+    fill_in "my-input", with: "hello"
+
+    sleep 1
```

Comment:
Hah, good catch! CI was failing consistently without the delay. Not sure the cause, maybe some queueing happening in the logger?